third_party/move/move-compiler/src/parser/ast.rs

```diff
@@ -1274,13 +1301,13 @@ impl AstDebug for StructDefinition {
             false
         });
 
-        if let StructFields::Native(_) = fields {
+        if let StructLayout::Native(_) = fields {
             w.write("native ");
         }
 
         w.write(&format!("struct {}", name));
         type_parameters.ast_debug(w);
-        if let StructFields::Defined(fields) = fields {
+        if let StructLayout::Singleton(fields) = fields {
```

Comment:
Ahh, those ast debug printers, I don't know of any use case, and they are already incomplete since long (e.g. abilities and attributes). Added a case now but it just prints variants are NYI for printing.